sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/handler/SendLinkHandler.java

```diff
@@ -21,10 +19,8 @@
     private final String linkName;
     private final String entityPath;
     private final AtomicBoolean isFirstFlow = new AtomicBoolean(true);
-    private final UnicastProcessor<Integer> creditProcessor = UnicastProcessor.create();
-    private final DirectProcessor<Delivery> deliveryProcessor = DirectProcessor.create();
-    private final FluxSink<Integer> creditSink = creditProcessor.sink();
-    private final FluxSink<Delivery> deliverySink = deliveryProcessor.sink();
+    private final Sinks.Many<Integer> creditProcessor = Sinks.many().unicast().onBackpressureBuffer();
+    private final Sinks.Many<Delivery> deliveryProcessor = Sinks.many().multicast().onBackpressureBuffer();
```

Comment:
Based on https://projectreactor.io/docs/core/release/api/reactor/core/publisher/DirectProcessor.html the closest Sinks equivalent would be `Sinks.many().multicast().directBestEffort()`